python/requirements.txt

```diff
@@ -9,3 +9,4 @@ Twisted>=12.1
 bitstring
 jsonschema
 cryptography>=1.0
+pyopenssl
```

Comment:
This seems unrelated?

Reply:
Yeah, I forgot to mention in the PR that there are two commits here -- this is needed to restore the Travis build to green.

Reply:
Might want it as a separate PR?

Reply:
Then this PR will not be mergable on its own, as Travis will be red.

(I will be doing rebase-and-merge not squash-and-merge, so it will remain a separate commit.)

Reply:
I meant just this commit to `python/requirements.txt`, as its own PR, which would restore the Travis build. Then, rebase this PR on top of the now-green tree, success?

Reply:
Yeah, that would ideally make sense but it would involve 3 x Travis runs (this PR before=fail, other PR, this PR after=success) rather than one, which I wanted to avoid.